.gitignore

```diff
@@ -12,3 +12,4 @@ pom.xml.asc
 .idea
 build.sh
 build
+fluree.cli.iml
```

Comment:
Not a huge deal but I like to ignore editor-specific files in my `~/.gitignore_global` file so that git never offers to add them to any of my repos (and conversely our repos don't have to explicitly ignore all the different editors' detritus). Details on enabling that here (though that author names his file `~/.gitignore` which I find a little more confusing but totally up to you): https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Not sure why I did this... I actually already have it excluded in my global. It was already check in though previously, and I was trying to get rid of it.